.github/CONTRIBUTING.md

```diff
@@ -48,6 +79,8 @@ To release a major you need to add `BREAKING CHANGE: ` to the start of the body
 
 Ensure that your code adheres to the included `.eslintrc` config by running `npm run test:checks`.
 
+Check your code is formatted by running `prettier` e.g. `npx prettier --check snykTask/**/*.ts` and auto-fix any violations by performing `npx prettier --write snykTask/**/*.ts`
```

Comment:
There're dedicated scripts for this already in package.json.

- `npm run test:checks` runs prettier verification (--check), as stated in the previous sentence
- `npm run format` runs formatter (--write)